xdsl/interpreters/riscv_scf.py

```diff
@@ -0,0 +1,39 @@
+from typing import Any
+
+from xdsl.dialects import riscv_scf
+from xdsl.interpreter import (
+    Interpreter,
+    InterpreterFunctions,
+    ReturnedValues,
+    impl,
+    impl_terminator,
+    register_impls,
+)
+
+
+@register_impls
+class RiscvScfFunctions(InterpreterFunctions):
+    bitwidth = 32
+
+    @impl(riscv_scf.ForOp)
+    def run_for(
+        self,
+        interpreter: Interpreter,
+        op: riscv_scf.ForOp,
+        args: tuple[Any, ...],
+    ):
+        lb, ub, step, *loop_args = args
+        loop_args = tuple(loop_args)
```

Comment:
I'm not sure this is redundant?

Reply:
the lhs is a tuple, the rhs is a list, this was to make pyright happy, otherwise the variable has type list | tuple

Reply:
I see, forget that `*loop_args` returns a list instead of tuple.